### PR TITLE
[MIG] membership_initial_fee: Migration to 12.0

### PR DESCRIPTION
**Pull request by chienandalu**
Charge an initial fee when a partner is invoiced for the first time with a
member product.

cc @Tecnativa

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]

**Review by amcor — APPROVED** [review]

**Comment by rafaelbn** [comment]
@chienandalu we should improve that the name of the product or the description of the product is in the description of the line in the invoice

**Review by rafaelbn — COMMENTED** [review]
Tested in runbot it works but we should improve about the description in the invoice lines

**Comment by chienandalu** [comment]
@rafaelbn I just added your suggestion, so you can either have the same default description as before or edit the fee product sale description to have a custom one.

File: membership_initial_fee/README.rst [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+* Tecnativa
+
+Contributors
+~~~~~~~~~~~~
+
+* `Tecnativa <https://www.tecnativa.com>`__:
+
+  * Pedro M. Baeza
+  * David Vidal
```
Comment (rafaelbn) [inline membership_initial_fee/README.rst:79]: Well after reviewing this modules from v8, v10, v11, v12..... I think this is contribution

**Review by rafaelbn — APPROVED** [review]
Tested functionally in runbot :+1:

**Pull request merged**